README.md

```diff
@@ -1,10 +1,11 @@
 # GitOps Toolkit
 
+(Check out our [Upcoming Meetups](#upcoming-meetups)!)
```

Comment:
Please remove this line

Reply:
I removed it. But is there a way to put *something* on the top half of the page that directs people downward to the talks/meetups? We want people to be able to see these things before getting lost in a wall of text...afterall they're informative, educational videos, not just marketing "fluff."

(Check out our [Upcoming Meetups](#upcoming-meetups)!)